tests/retrieve-secrets/start

```diff
@@ -6,7 +6,6 @@ cd $(dirname $0)
 
 echo "Building mock-conjur-env..."
 
-[ -d "temp_clone_dir" ] && rm -rf "temp_clone_dir"
 mkdir temp_clone_dir
```

Comment:
The `mkdir temp_clone_dir` will fail if this is run a second time. Maybe better to check if the `temp_clone_dir` exists, and if it doesn't yet exist, then do the `mkdir` and `git clone ...` commands?

Reply:
Added `-p` flag to `mkdir`